package.json

```diff
@@ -4,7 +4,7 @@
         "type": "ssh",
         "url": "https://github.com/guardian/atoms-rendering"
     },
-    "version": "2.4.1",
+    "version": "3.0.0-0",
```

Comment:
Was this change intentional?

Reply:
Nope, I was trying to do a secret alpha release and forgot there was a prerelease script thing.